### PR TITLE
Add Grafana-client recipe

### PR DESCRIPTION
**Pull request by Marie59**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Marie59** [comment]
@conda-forge/help-python this recipe is ready ! Thanks

File: recipes/grafana-client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools-scm
```
Comment (BastianZim) [inline recipes/grafana-client/meta.yaml:20]:
```suggestion
```

File: recipes/grafana-client/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/grafana-client-{{ version }}.tar.gz
+  sha256: 7ee2f865dbfc89d3eb8bc9df41417ef7c1c905f9d92336bce0af27fb37607bc6
+
+build:
+  skip: true  # [py2k]
```
Comment (BastianZim) [inline recipes/grafana-client/meta.yaml:13]:
```suggestion
  noarch: python
```

File: recipes/grafana-client/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
```
Comment (BastianZim) [inline recipes/grafana-client/meta.yaml:19]:
```suggestion
    - python >=3.6
```

File: recipes/grafana-client/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    - setuptools-scm
+    - setuptools-scm <8
+    - pip
+  run:
+    - python
```
Comment (BastianZim) [inline recipes/grafana-client/meta.yaml:24]:
```suggestion
    - python >=3.6
```

File: recipes/grafana-client/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    - requests >=2.23.0,<3
+    - dataclasses  # [py<=36]
```
Comment (BastianZim) [inline recipes/grafana-client/meta.yaml:26]:
```suggestion
    - dataclasses
```
Ref: https://conda-forge.org/docs/maintainer/knowledge_base.html#empty-python-packages

File: recipes/grafana-client/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    - importlib-metadata  # [py<=37]
```
Comment (BastianZim) [inline recipes/grafana-client/meta.yaml:27]:
```suggestion
    - importlib-metadata
```

**Review by BastianZim — CHANGES_REQUESTED** [review]
This can probably be noarch

**Comment by BastianZim** [comment]
@conda-forge-admin, please lint

**Pull request merged**